src/test/java/edu/harvard/iq/dataverse/settings/JvmSettingsTest.java

```diff
@@ -21,11 +21,17 @@ void lookupSetting() {
     void lookupPidProviderSetting() {
         assertEquals("test", JvmSettings.DATACITE_USERNAME.lookup("datacite"));
     }
-    
+
```

Comment:
I'm not sure this adds anything/suggest removing. The DataCite example is nominally just a test of the lookup(param) call and this just repeats that with another setting.

Reply:
I'll remove it